src/terraform/commands.go

```diff
@@ -10,6 +10,9 @@ import (
 
 // RunInit runs the 'terraform init' command and force copies the remote state
 func RunInit(deployConfig types.DeployConfig) error {
+	if !util.DoesFileExist(deployConfig.TerraformFile) {
+		return errors.New("terraform initialization attempted in directory with no 'main.tf' file. Please make sure 'exo deploy build' has been run")
+	}
```

Comment:
When the user makes changes to the application then then `exo deploy build` needs to be run again so `main.tf` can be updated, so that deployment will reflect the latest version of the app.